javascript/javascript1/week4/voiceassistant.js

```diff
@@ -0,0 +1,72 @@
+const Name = [];
+const toDoList = [];
+
+function getReply(command) {
+  for (i = 0; i < Name.length; i++);
+
+  const NewName = "benjamin";
+  Name.push(NewName);
+
+  if (command.includes("benjamin")) {
+    console.log(`hi my name is ${NewName}`);
+    return `hello  ${NewName} nice to meet you`;
+  } else if (command === "what is my name?") {
+    const anotherName = "claudia";
+    Name.push(anotherName);
+    return `my name is  ${anotherName}`;
+  }
+
+  if (command === "fishing") {
```

Comment:
fishing is only an example, the purpose is that when you get the command of `add something to my todo`, you take that out of sentence and add it to the todo.